### PR TITLE
fix truncating conversion warning

### PR DESCRIPTION
**Pull request by evenex**
fixed with explicit cast to assignment target type

### CODE REVIEW

**Comment by Dgame** [comment]
Which version do you use?  `op=` never did throw an error before.

File: source/Dgame/System/StopWatch.d [hunk 1]
```diff
@@ -138,7 +138,7 @@ struct Time {
         time.minutes -= floor(float(time.hours)) * 60;
         time.minutes = floor(time.minutes);
         time.seconds -= floor(min) * 60;
-        time.msecs -= floor(sec) * 1000;
+        time.msecs -= cast(typeof(time.msecs))(floor(sec) * 1000);
```
Comment (Dgame) [inline source/Dgame/System/StopWatch.d]: I would prefer `cast(float)`. But I can change that by myself if you don't want to do that. :)

**Comment by evenex** [comment]
I'm using the latest DMD from git HEAD. Anyways I can change it when I get home tonight or you can change it now, whichever is fine

**Comment by evenex** [comment]
Updated

**Comment by Dgame** [comment]
Thank you.

**Pull request merged**